bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define wdec = u c d s u~ c~ d~ s~ l+ l- vl vl~
+import model HC_NLO_X0_UFO
+generate p p > x0 t t~, (t > b w+, w+ > wdec wdec), (t~ > b~ w-, w- > wdec wdec)
```

Comment:
same comment on w > tau

Reply:
defined

Reply:
I guess using decay chain will make this gridpack large, if it's case you can move to madspin for top decay